bundles/org.openhab.binding.max/src/main/resources/OH-INF/i18n/max.properties

```
@@ -130,8 +130,8 @@ thing-type.config.max.thermostatplus.ecoTemp.label = Eco Temperature
 thing-type.config.max.thermostatplus.ecoTemp.description = Set the Eco Temperature.
 thing-type.config.max.thermostatplus.group.actions.label = Actions
 thing-type.config.max.thermostatplus.group.actions.description = Action Buttons
-thing-type.config.max.thermostatplus.group.binding.label = Binding Settings
-thing-type.config.max.thermostatplus.group.binding.description = Binding settings for this device
+thing-type.config.max.thermostatplus.group.addon.label = Binding Settings
```

And this.

---

Not sure what happened here, but still wrong.
```suggestion
thing-type.config.max.thermostatplus.binding.addon.label = Binding Settings
```